src/SFA.DAS.LevyTransferMatching.Web/Controllers/SearchFundingController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using SFA.DAS.LevyTransferMatching.Web.Models.SearchFunding;
+using SFA.DAS.LevyTransferMatching.Web.Orchestrators;
+
+namespace SFA.DAS.LevyTransferMatching.Web.Controllers
+{
+    [AllowAnonymous]
+    [Route("search-funding")]
+    public class SearchFundingController : Controller
+    {
+        private readonly ISearchFundingOrchestrator _searchFundingOrchestrator;
+
+        public SearchFundingController(ISearchFundingOrchestrator searchFundingOrchestrator)
+        {
+            _searchFundingOrchestrator = searchFundingOrchestrator;
+        }
+
+        public async Task<IActionResult> Index()
+        {
+            var viewModel = await _searchFundingOrchestrator.GetSearchFundingViewModel();
```

Comment:
Pedantic, but since this is the Index controller action and the Index view, we should probably call the viewmodel IndexViewModel and rename the method to suit - it will exist in a different namespace to the other index, so should be fine.